packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -1413,6 +1413,26 @@ Expected input body:
 { "property": { "property": "hello" } }
 ```
 
+### Models_Property_Types_NullableModel_get
+
+- Endpoint: `get /models/properties/types/nullable-model`
+
+Expected response body:
+
+```json
+{ "property": null }
+```
+
+### Models_Property_Types_NullableModel_put
+
+- Endpoint: `put /models/properties/types/nullable-model`
+
+Expected input body:
+
+```json
+{ "property": null }
```

Comment:
Personally, as REST API, I don't think PUT `{ "property": null }` semantically be any different from a PUT with `{}`.

But I won't block the case.

Reply:
Will JS treat '{}' as undefined and 'null' as null?

Reply:
Let me change the test case to JSON MERGE PATCH with partial update - https://www.rfc-editor.org/rfc/rfc7386#section-1.

```
{
         "a":"z",
         "c": {
       "f": null
         }
       }

```

Reply:
Yes, json merge patch would be great!